test/models/requests/create_token_request_test.dart

```diff
@@ -4,7 +4,7 @@ import 'package:omise_dart/src/models/requests/create_token_request.dart';
 void main() {
   group('CreateTokenRequest', () {
     test('toJson should convert CreateTokenRequest to correct map', () {
-      const request = CreateTokenRequest(
+      final request = CreateTokenRequest(
```

Comment:
I am curious why are we changing `const` to `final`?

Reply:
when I made the model mutable it can't be `const` anymore.